3wifi.sql

```diff
@@ -36,7 +36,7 @@ CREATE TABLE `base` (
 	INDEX `ESSID` (`ESSID`),
 	INDEX `Time` (`time`),
 	UNIQUE INDEX `WIFI` (`NoBSSID`, `BSSID`, `ESSID`, `WiFiKey`, `WPSPIN`)
-) COLLATE='utf8_general_ci' ENGINE=InnoDB ROW_FORMAT=FIXED;
+) COLLATE='utf8_general_ci' ENGINE=InnoDB ;
```

Comment:
Everything looks ok!

I'd suggest to remove that trailing space between InnoDB and semicolon.